libraries/Microsoft.Bot.Schema/Teams/TeamsMeetingMember.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Data about the joined meeting participants.
```

Comment:
Is not just the joined participant. This class is used in both join and leave events. Review all the comments.

Reply:
This feedback wasn't applied.